Assets/SEE/UI/SettingsMenu.cs

```diff
@@ -0,0 +1,69 @@
+using UnityEngine.UI;
+using UnityEngine;
+using TMPro;
+using SEE.Controls;
+using SEE.Utils;
+using SEE.GO;
+
+namespace SEE.UI
+{
+    /// <summary>
+    /// Handles the actions in the
+    /// settings prefab.
+    /// </summary>
+    public class SettingsMenu : PlatformDependentComponent
+    {
+        /// <summary>
+        /// Prefab for the SettingsMenu.
+        /// </summary>
+        protected virtual string SettingsPrefab => UIPrefabFolder + "SettingsMenu";
+
+        /// <summary>
+        /// The SettingsMenu game object.
```

Comment:
Use `<see cref="SettingsMenu"/>` when referring to declarations within comments. This way, rename refactorings will also update your comment.

Reply:
I fixed it.